content/docs/CLI/source-git/init.md

```diff
@@ -1,94 +1,124 @@
 ---
-title: "init"
+title: "packit source-git init"
 draft: false
 disableToc: false
 weight: 1
 ---
 
-# source-git init
+# `packit source-git init`
```

Comment:
I'm not against it, but I prefer consistency so if you want this, please do the same change on all pages.

Reply:
oki, that's a good point, I'll do that